src/matching_market.sol

```diff
@@ -17,23 +17,19 @@
 
 pragma solidity ^0.5.12;
 
-import "./expiring_market.sol";
 import "ds-note/note.sol";
+import "./simple_market.sol";
+import "./oracle/PriceOracle.sol";
 
 contract MatchingEvents {
-    event LogBuyEnabled(bool isEnabled);
     event LogMinSell(address pay_gem, uint min_amount);
-    event LogMatchingEnabled(bool isEnabled);
     event LogUnsortedOffer(uint id);
     event LogSortedOffer(uint id);
     event LogInsert(address keeper, uint id);
     event LogDelete(address keeper, uint id);
 }
 
-contract MatchingMarket is MatchingEvents, ExpiringMarket, DSNote {
-    bool public buyEnabled = true;      //buy enabled
-    bool public matchingEnabled = true; //true: enable matching,
-                                         //false: revert to expiring market
+contract MatchingMarket is MatchingEvents, SimpleMarket, DSNote {
```

Comment:
`DSNote` can be completely removed (including the dependency). It's not being used anywhere.

Reply:
Done in my PR.